.github/workflows/build_wheels_linux.yml

```diff
@@ -167,21 +167,21 @@ jobs:
           cache-key: ${{ inputs.cache-key }}
           repository: ${{ inputs.repository  }}
           script: ${{ inputs.pre-script }}
-      - name: Build clean
-        working-directory: ${{ inputs.repository }}
-        shell: bash -l {0}
+      - name: Install python build dependency
         run: |
           set -euxo pipefail
+          # shellcheck disable=SC1090
           source "${BUILD_ENV_FILE}"
-          ${CONDA_RUN} python setup.py clean
+          # shellcheck disable=SC2086
+          ${CONDA_RUN} python -m pip install build
       - name: Build the wheel (bdist_wheel)
         working-directory: ${{ inputs.repository }}
         shell: bash -l {0}
         run: |
           set -euxo pipefail
           source "${BUILD_ENV_FILE}"
           export PYTORCH_VERSION="$(${CONDA_RUN} pip show torch | grep ^Version: | sed 's/Version:  *//' | sed 's/+.\+//')"
-          ${CONDA_RUN} python setup.py bdist_wheel
+          ${CONDA_RUN} python -m build --wheel -n
```

Comment:
`-n` stands for "--no-isolation", which we use b/c we already setup a conda environment.

Reply:
Nitpick: should use long form "--no-isolation" instead of "-n" for readability,